src/components/aeDivider/aeDivider.js

```diff
@@ -0,0 +1,27 @@
+const aeDividerTypes = [
+  'boring',
+  'normal',
```

Comment:
I propose to remove `normal` from divider types since it does nothing. Just apply corresponding background color by default to this component.